util/encoding/key_encoding_test.go

```diff
@@ -291,13 +294,76 @@ func disabledTestFloatMandE(t *testing.T) {
 		{1234.5, 2, []byte{0x19, 0x45, 0x64}},
 		{12.345, 1, []byte{0x19, 0x45, 0x64}},
 		{0.123, 0, []byte{0x19, 0x3c}},
+		{-0.123, 0, []byte{0x19, 0x3c}},
```

Comment:
GIven that `floatMandE` takes the absolute value as its first operation, testing negative values isn't terribly interesting. I've removed this test case.